src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PossibleDeNovo.java

```diff
@@ -35,6 +34,21 @@
  */
 public final class PossibleDeNovo extends InfoFieldAnnotation {
 
+    public PossibleDeNovo(final Set<Trio> trios, final double minGenotypeQualityP) {
```

Comment:
Mention `PossibleDeNovo` in https://github.com/broadinstitute/gatk/issues/1468 as one of the annotations that is not fully usable until it is resolved.